docs/blog/2021-10-18-announcing-lunasec.md

```diff
@@ -28,7 +28,7 @@ offering enterprise and open-source support for LunaDefend.
 
 AWS, NodeJS, and React is the fully supported stack at the moment, and now we are hard at work adding support for more environments.
 
-Please take a moment to learn about LunaDefend by reading the rest of this post and trying the [example app](https://www.lunasec.io/docs/pages/overview/demo-app/overview/) (it's really easy).
+Please take a moment to learn about LunaDefend by reading the rest of this post and trying the [example app](https://www.lunasec.io/docs](/pages/lunadefend/overview/demo-app/overview/) (it's really easy).
```

Comment:
It looks like you've got a typo everywhere you replaced this. You don't need the `](` between `docs` and `/pages`. You just need it to be `https://www.lunasec.io/docs/pages/lunadefend/overview/demo-app/overview/` :ok_hand:

Reply:
Link should be fixed now